thirdparty/vars.sh

```diff
@@ -375,6 +375,14 @@ BENCHMARK_NAME=benchmark-1.5.6.tar.gz
 BENCHMARK_SOURCE=benchmark-1.5.6
 BENCHMARK_MD5SUM="668b9e10d8b0795e5d461894db18db3c"
 
+# breakpad
+# breakpad has no release version, the source is from commit@38ee0be,
+# and also add lss files. See README.md in it.
+BREAKPAD_DOWNLOAD="https://doris-thirdparty-repo.bj.bcebos.com/thirdparty/breakpad-src-38ee0be-with-lss.tar.gz"
```

Comment:
you'd better use 
 an official download link https://github.com/google/breakpad/archive/38ee0be4d1118c9d86c0ffab25c6c521ff99fdee.zip 
and use bos as a mirror

Reply:
This is because:
1. breakpad has no official release version, so i have to use trunk code with a certain commit.
2. we need to add lss files before compiling it from source code, which i hava already added to the tarball and noticed in README.md